### PR TITLE
RGB LED Revision

### PR DESCRIPTION
**Pull request by jamesjpenguin**
Added KiCAD design for RGB LEDs added to each button.

Added Readme and images as well

### CODE REVIEW

**Comment by mizma** [comment]
oh, didn't see this one.  let me review the files this weekend.  thanks for the cool PR!

File: hardware-rgb/.DS_Store
Comment (mizma) [inline hardware-rgb/.DS_Store:1]: could you add .gitignore for all .DS_Store files and remove them?  these are unnecessary in the repo (Mac specific file options storage).
thanks.

File: hardware-rgb/pcb/ergoSHIFT-rgb-bottom-pos.csv
Comment (mizma) [inline hardware-rgb/pcb/ergoSHIFT-rgb-bottom-pos.csv:1]: Is this file needed?

Comment (jamesjpenguin) [reply hardware-rgb/pcb/ergoSHIFT-rgb-bottom-pos.csv:1]: nope! removed

File: hardware-rgb/.DS_Store
Comment (jamesjpenguin) [reply hardware-rgb/.DS_Store:1]: updated .gitignore, removed .DS_Store, removed extra gerber files where I added a personal logo to my boards, updated Kicad PCB silkscreen to also remove personal logo

**Comment by jamesjpenguin** [comment]
Sorry for the excessive commits, I'm relatively new to git and contributing to others' repos!

**Comment by mizma** [comment]
No problem!  I will download the data and check the data this weekend and will merge if I don't see any issues.
Thanks

**Comment by mizma** [comment]
I've got around to checking the kicad data as well.  One thing I would like is to clarify that the RGB variant is not southpaw compatible.  Since the Readme refers to the rev2 build instructions, I think it would be important to add a note in the readme that this rgb version cannot be reversed to make the southpaw version.
As a related note, it might be good to remove the silkscreen marks on the backside (the button legend and maybe the logo on the corner) since they are there for the southpaw build.

Other thant that, I think the design looks great!

**Comment by jamesjpenguin** [comment]
Got  it, will make the updates and commit again when I have a chance!

**Comment by jamesjpenguin** [comment]
Sorry again for the extra commits, I got my remote and local a bit mixed up, but should have it cleaned up by now.

**Comment by mizma** [comment]
looks good! thanks.
